client/my-sites/customer-home/cards/features/stats/index.jsx

```diff
@@ -1,136 +1,293 @@
 /**
```

Comment:
Another rename here, so feel free to ignore (previous `stats` has been deleted, and `stats-v2` has been renamed to `stats`).